### DISCUSSION TITLE
Community Moderation App

### DISCUSSION THREAD

**Discussion by BenLloydPearson** [mattermost/community#26] [Practical Ideas]
Larger communities, such as Rocky Linux, have shown interest in using Mattermost. However, one limitation larger communities often face is the need for moderation tools that enable them to enforce community standards and codes of conduct. Other than the custom permissions that are only available in paid versions of Mattermost, we don't have a lot of capabilities to help our users moderate their server. 

I would love to see a moderation app that provides at least some basic moderation capabilities, such as:
* The ability for any user to flag inappropriate messages from other users
* A page to view open and closed moderation requests and take action (ignore, remove post, ban, etc.)
*  A permissions scheme to let server admins appoint moderator roles that can ban users, remove posts, and see private messages that are flagged.

There are probably tons of other things we could add to this, but these features would get an MVP up and running.

**Comment by emilyacook** [comment]
10000%, I've been thinking about how to do this for us internally. I think there might be potential to use incident collaboration for point 2?

**Reply by BenLloydPearson** [reply]
Oh! Incident collaboration is a great idea.

**Comment by chetanyakan** [comment]
Can also implement using: https://github.com/mattermost/community/discussions/95
